cmlreaders/readers/eeg.py

```diff
@@ -146,20 +147,29 @@ def __init__(self, filename: str, dtype: Type[np.dtype],
         self.rereferencing_possible = True
 
     @abstractmethod
-    def read(self) -> np.ndarray:
+    def read(self) -> Tuple[np.ndarray, List[int]]:
```

Comment:
any reason you elected not to use a NamedTuple? I thought that was your preferred method for functions that have multiple return values

Reply:
`read` is meant to be a completely internal method so I don't think it's as important in this case.